lib/vmdb/settings.rb

```diff
@@ -189,6 +188,22 @@ def self.reset_settings_constant(settings)
     end
     private_class_method :reset_settings_constant
 
+    def self.replace_magic_values!(settings, resource)
+      parent_settings = template_settings = nil
+
+      walk(settings) do |key, value, path, owner|
+        case value
+        when RESET_COMMAND
+          parent_settings ||= parent_settings_without_local(resource).load!.to_hash
+          owner[key] = parent_settings.fetch_path(*path)
```

Comment:
This passes all existing tests for the reset command, though I think a few more tests are needed.